apps/sms/services/js/messaging/messaging_client.js

```diff
@@ -0,0 +1,50 @@
+/* global bridge
+*/
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for messaging service.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'messaging-service';
+
+  /**
+   * Reference to active bridge client instance.
+   *
+   * @type {Client}
+   */
+  var client;
+
+  var MessagingClient = {
+    init() {
+      var serviceEndpoint = new SharedWorker(
+        '/services/js/messaging/messaging_service.js'
+      );
+
+      serviceEndpoint.port.start();
+
+      client = bridge.client(SERVICE_NAME, serviceEndpoint.port);
+    },
+
+    sendSMS(options) {
+      return client.method('sendSMS', options);
+    },
+
+    sendMMS(options) {
+      return client.method('sendMMS', options);
+    },
+
+    resendMessage(message) {
+      return client.method('resendMessage', message);
+    },
+
+    retrieveMMS(id) {
+      return client.method('retrieveMMS', id);
+    }
```

Comment:
wondering if we should make this more automatic; I realize some client methods could be more complex, but we'll have a lot of simple methods like those. Either a forEach, or using a ES6 proxy ? Not urgent and not necessary in this patch, just thinking out loud and asking what you think about this.